<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_eleve</class>
 <widget class="QDialog" name="gestion_eleve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>731</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des eleves</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_sexe">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>199</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Homme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femme</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="le_age">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>350</y>
       <width>199</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>122</y>
       <width>199</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="error5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>199</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>480</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="error2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prenom">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>174</y>
       <width>199</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="error3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="error6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pr√©nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Date naissance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Classe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="error1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="le_date">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>199</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="error4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>440</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_qrcode">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>370</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="qrcodegen">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>320</y>
       <width>200</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>QR Code</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tri">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom (A-&gt;Z)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prenom (A-&gt;Z)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>daten (des)</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recherche">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Afficher">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_supp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>440</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="qrcodecommande">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>90</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="statistique">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
     </property>
     <property name="text">
      <string>Statistique</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>631</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Historique</string>
    </attribute>
    <widget class="QTableView" name="tableView_historique">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>981</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>CAMERA</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>441</width>
       <height>361</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>359</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="opcionesPushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>441</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
